WcaOnRails/config/locales/en.yml

```diff
@@ -714,10 +727,14 @@ en:
       competition_was: "Competition was %{n_days} ago."
     #context: in the "nearby competitions" list
     nearby_competitions:
-      comp: "competition"
+      #context: pluralization of "x competition(s)" in the sentence "x competition(s) within n km and m days."
+      competitions:
```

Comment:
This key (and the others that went from a "single" key to a "pluralizable" key), was renamed to avoid a bug in Jonatan's app.